sdks/python/gen_xlang_wrappers.py

```diff
@@ -18,6 +18,8 @@
 """
 Generates Python wrappers for external transforms (specifically,
 SchemaTransforms)
+
+Experimental; no backwards compatibility guarantees.
```

Comment:
we should try to have backwards compatibility on the wrappers we generate though.